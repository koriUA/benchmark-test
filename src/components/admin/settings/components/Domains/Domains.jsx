import React from 'react';

const Domains = () => <div>Here goes content for Domains</div>;

export default Domains;
