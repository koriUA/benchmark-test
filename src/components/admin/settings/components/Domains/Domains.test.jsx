import React from 'react';
import { mount } from 'enzyme';

import Domains from './Domains';

describe('Content', () => {
  test('should render the Domains', () => {
    const component = mount(<Domains />);
    expect(component).toMatchSnapshot();
  });
});
