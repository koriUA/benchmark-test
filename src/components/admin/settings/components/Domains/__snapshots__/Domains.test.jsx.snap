// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render the Domains 1`] = `
<Domains>
  <div>
    Here goes content for Domains
  </div>
</Domains>
`;
